stable/logstash/templates/NOTES.txt

```diff
@@ -1,21 +1,12 @@
-{{- if .Values.service.ports.http }}
-Get the Logstash HTTP Input URL by running these commands:
```

Comment:
The previous notes seemed really specific to accessing the cluster through whatever port was dedicated for the HTTP plugin. I've removed most of those instructions since it doesn't feel relevant anymore and restrictive. In the future though I'd like to modify the ingress template to use the exposed ports so events can be sent to Logstash from outside of the cluster.